notebooks/daniel_notebook.ipynb

```diff
@@ -1,191 +1,26 @@
 {
```

Comment:
<p>This will make output a little nicer:</p><p><code>&gt;&gt;&gt; print('forceA = ', forceA)</code> </p><p><code>forceA = [1 0 0]</code></p><p> </p><p>A tutorial guideline from Google Style: write in the second-person e.g. </p><p> </p><p>"Use matplotlib to visualize these force _vectors_"</p><p> </p><p>"You can add them together"</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cooperrc/uconn_numpy_tutorials/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>